components/Habit.tsx

```diff
@@ -62,17 +70,35 @@ type HabitProps = {
   streak: number;
   habit: THabit;
   entries: Entry[];
+  addHabitEntry: () => void;
```

Comment:
I recommend naming these with an event-style convention: `onRemoveEntry`, `onY`, since the parent component is passing in functions that are to be called _when_ something happens

Reply:
Just to clarify: should the parent then also have its functioned name as `handleAddEntry` instead of `add...`?

Reply:
Both are perfectly acceptable!